covid-19-support/src/components/Sidebar.vue

```diff
@@ -0,0 +1,221 @@
+          >{{ $t('sidebar.info-end-text') }}
+        </InfoPanel>
+
+        <InfoPanel :infotype="'handwash'" :icon="'fa-hands-wash'" v-if="filteredMarkers.length == 0">
+          <b class="themeFont">{{ $t('sidebar.shopsafe') }}</b>
+          <br />
+          (1) {{ $t('sidebar.stayhome') }}<br />
+          (2) {{ $t('sidebar.sixfeet') }}<br />
+          (3) {{ $t('sidebar.washhands') }}<br />
+        </InfoPanel>
+      </div>
+
+      <BusinessDetails
+        :infotype="'green'"
+        :icon="'fa-tractor'"
+        :business="location.currentBusiness"
+        v-if="location.currentBusiness != null && showListing != true"
+        @close-details="closeDetails"
+      ></BusinessDetails>
+    </div>
+
+    <results-list
+      :filteredMarkers="highlightFilteredMarkers"
+      :location="location"
+      @location-selected="passLocation"
+      v-if="showListing"
+      :selected-day="day"
+    />
+  </div>
+</template>
+
+<script>
+import BusinessDetails from './BusinessDetails.vue'
+import InfoPanel from './InfoPanel.vue'
+import ResultsList from './ResultsList.vue'
+import SearchFilters from './SearchFilters.vue'
+
+export default {
+  name: 'sidebar',
+  components: {
+    BusinessDetails,
+    InfoPanel,
+    ResultsList,
+    SearchFilters
+  },
+  data() {
+    return {
+      locationData: location,
+      showListing: this.showList
+    }
+  },
+  props: {
+    isFilterOpen: Boolean,
+    need: String,
+    day: Number,
+    filteredMarkers: Array,
+    highlightFilteredMarkers: Array,
+    location: { locValue: Number, locId: String, isSetByMap: Boolean, currentBusiness: Object },
+    showList: Boolean
+  },
+  computed: {
+    tabtitle() {
+      return this.isFilterOpen ? this.$t('sidebar.close-panel') : this.$t('sidebar.open-panel')
+    }
+  },
+  methods: {
+    closeDetails: function () {
+      this.showListing = true
+    },
+    passLocation: function (val) {
+      this.locationData = val
+      this.showListing = false
+      this.$emit('location-selected', val)
+    }
+  },
+  watch: {
+    day: function () {
+      this.locationData = null
+      this.showListing = true
+    },
+    need: function (val) {
+      this.locationData = null
+      if (val == 'none') {
+        this.showListing = false
+      } else {
+        this.showListing = true
+      }
+    },
+    location: function () {
+      this.showListing = false
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.custom-select {
+  font-size: 0.8rem;
+  cursor: pointer;
+}
+
+#sidebar-wrapper {
+  transition: transform 0.25s ease-out;
+  transform: translateX(-100%);
+  z-index: 1035;
+  max-height: 100vh;
+  width: 100%;
+  height: 100vh;
+  background: theme-color('secondary');
+
+  @media (prefers-color-scheme: dark) {
+    background: theme-color('secondaryDark');
+  }
+
+  @include media-breakpoint-up(md) {
+    height: auto;
```

Comment:
If you add the `transform: translateX(-100%);` here, then everything should still work correctly on tablet & desktop. 

I.e. it will continue to slide in and out from the right hand side.